src/NextGen/DonationForm/ViewModels/DonationFormViewModel.php

```diff
@@ -252,6 +251,8 @@ private function enqueueFormScripts(int $formId, string $formDesignId)
             'give'
         ))->loadInFooter()->enqueue();
 
+        Hooks::doAction('givewp_form_view_enqueue_scripts');
```

Comment:
@pauloiankoski this location looks good, but let's change this to:

```suggestion
        Hooks::doAction('givewp_donation_form_enqueue_scripts');
```